ddtrace/contrib/flask/patch.py

```diff
@@ -71,11 +73,21 @@
 )
 
 
-def get_version():
```

Comment:
Can we revert this change? get_version is defined every integration and is used in ddtrace._monkey. Renaming this method would require a significant change that will updated tens of files.